src/storage/store/engine.rs

```diff
@@ -641,17 +645,21 @@ impl ShardEngine {
                                                 events.push(hub_event.clone());
                                             }
                                             Err(err) => {
-                                                warn!(
-                                                    "Error revoking existing username: {:?}",
-                                                    err
-                                                );
+                                                if source != ProposalSource::Simulate {
```

Comment:
Would've been cleaner to change the log level dynamically. But it requires some rust macro magic. Just went with an if check for now. We can cleanup later.